kpet/run.py

```diff
@@ -18,26 +18,28 @@
 from kpet import utils
 
 
-def generate(args):
+def generate(template_content, kernel, output=None, arch='x86_64',
+             description=''):
     """Generate an xml output compatible with beaker"""
```

Comment:
Could you please document the function arguments? One of the benefits of having explicit arguments is that it's easier to document them :)

Reply:
Done